app/controllers/miq_policy_controller.rb

```diff
@@ -943,7 +943,9 @@ def folder_get_info(folder_node)
     @sb[:folder] = nodeid.nil? ? nodetype.split("-").last : nodeid
     if x_active_tree == :policy_tree
       if nodeid.nil? && ["compliance", "control"].include?(nodetype.split('-').last)
-        @folders = ["Host #{nodetype.split('-').last.titleize}", "Vm #{nodetype.split('-').last.titleize}"]
+        @folders = ["Host #{nodetype.split('-').last.titleize}", "Vm #{nodetype.split('-').last.titleize}",
+                    "Container Image #{nodetype.split('-').last.titleize}"]
```

Comment:
I've added:

```UI_FOLDERS = [Host, Vm, ContainerImage].freeze```

to MiqPolicy and then here:
```
MiqPolicy::UI_FOLDERS.collect do |model|
  "#{model.name.titleize} #{nodetype.split('-').last.titleize}"
end
```
and at policies.rb
```

MiqPolicy::UI_FOLDERS.collect(&:name)```
to replace the above and 
```